app/src/main/java/org/openimis/imispolicies/OverViewPolicies1.java

```diff
@@ -2,26 +2,24 @@
 
```

Comment:
We should change the name of this file to OverViewPolicies.java (without 1).

Reply:
Changed